crossbuilder

```diff
@@ -958,8 +1140,21 @@ check_command_parameter_count () {
 
 detect_package () {
     if test -e debian/changelog ; then
-        PACKAGE=`dpkg-parsechangelog --show-field Source`
-        PACKAGE_VERSION=`dpkg-parsechangelog --show-field Version`
+        if which dpkg-parsechangelog > /dev/null 2>&1 ; then
+            PACKAGE=`dpkg-parsechangelog --show-field Source`
+            PACKAGE_VERSION=`dpkg-parsechangelog --show-field Version`
+        else
+              # Get package name and version parsing the debian/changelog file using grep.
```

Comment:
there's something wrong with the indentation here

Reply:
done